### PR TITLE
Created posibility to gift certificate and send emails

### PR DESCRIPTION
**Pull request by MickhaelSh**
## Description

Created pugs for email and servise for changing email in certificate

reciveMail

sendMail

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date front-end and admin part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: email-templates/resive-gift-certificate-en.pug [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	div.contentWrapper 
+		p.text Greetings!#[br]You received a HORONDI certificate №!{certificateName} for #[strong !{certificateValue}] UAH as a gift from one of our users. You can view your certificates on our #[a(href=frontendUrl) website.]
+		img.certificate(src=`https://horondi.blob.core.windows.net/horondi/images/certificate-${certificateValue}.png`)
+		h2.productTitle.bestWishes Pay attention!
+		p.productMaterial 1. The gift certificate provides a discount on any Horondi product.
+		p.productMaterial 2. If the purchase price exceeds the available balance of the gift certificate, the difference can be paid in a convenient way.
+		p.productMaterial 3. The gift certificate is not subject to return or exchange for the corresponding cash equivalent.
+		p.productMaterial.bestWishes 4. The certificate is valid for 365 days from the moment of its purchase - #[strong !{dateEnd}] (expiration date of your certificate)
+		button.continue
+			a(href=frontendUrl + `/my-certificates/`) view my certificates
```
Comment (High-Voltaged) [inline email-templates/resive-gift-certificate-en.pug:16]: Please fix this no newline here.

File: email-templates/index.js [hunk 2]
```diff
@@ -55,6 +55,17 @@ const htmlTemplates = {
       'certificate-user-purchased-email-ua',
     ],
   },
+  [EmailActions.SEND_GIFT_CERTIFICATE]: {
+    subject: '[HORONDI] Certificate has been gifted',
+    templateFileName: ['send-gift-certificate-en', 'send-gift-certificate-ua'],
+  },
+  [EmailActions.RESIVE_GIFT_SERTIFICATE]: {
```
Comment (High-Voltaged) [inline email-templates/index.js:62]: I think you meant `RECEIVE`?

File: modules/certificate/certificate.service.js [hunk 3]
```diff
@@ -235,6 +239,31 @@ class CertificatesService extends FilterHelper {
...
+    });
+    await sendEmail(oldEmail, SEND_GIFT_CERTIFICATE, {
+      certificateName: certificate.name,
+      certificateValue: certificate.value,
+      dateStart,
+      dateEnd,
+      language,
+    });
+
+    return true;
```
Comment (High-Voltaged) [inline modules/certificate/certificate.service.js:264]: The graphql declaration says this function is supposed to return a `CertificateResult`, which is not just a boolean value. Why did you return this?

File: email-templates/index.js (hunk 2 above)
Comment (MickhaelSh) [reply email-templates/index.js]: fixed

File: modules/certificate/certificate.graphql.js [hunk 4]
```diff
@@ -73,6 +73,7 @@ const certificateTypes = `
     deleteCertificate(id: ID!): CertificateResult
     generateCertificate (newCertificates: [GenerateCertificateInput]!, email: String, dateStart: Date ): CertificatesResult
     updateCertificate(name: String!): CertificateResult
+	 gitftCertificateToEmail(id: ID!, email: String!, oldEmail: String!, language: Int!): CertificateResult
```
Comment (High-Voltaged) [inline modules/certificate/certificate.graphql.js:76]: Please fix the naming of `gitftCertificateToEmail` to `giftCertificateToEmail`

File: modules/certificate/certificate.service.js (hunk 3 above)
Comment (MickhaelSh) [reply modules/certificate/certificate.service.js:264]: fixed

File: modules/certificate/certificate.graphql.js (hunk 4 above)
Comment (MickhaelSh) [reply modules/certificate/certificate.graphql.js]: done

**Review by High-Voltaged — DISMISSED** [review]

**Review by dmytruk-y — DISMISSED** [review]

**Review by High-Voltaged — APPROVED** [review]

**Review by dmytruk-y — APPROVED** [review]

**Pull request merged**